campaigns/mofo-eoy-2016/snippet.html

```diff
@@ -0,0 +1,328 @@
+{#
+2016 fundraising snippet.
+
+Variables:
+
+  @text: main text - Text of snippet.
+  @icon: image - Snippet icon. 64x64px. SVG or PNG preferred.
+  @button_label: Required. Text for donate button.
+  @donation_form_url: Required. Url to the donation form.
+
+  @button_color: Default blue. Red and green also options.
+
+  @locale: Default en-US. String for the locale code.
+  @currency_code: Default usd. The code for the currency. Examle gbp, cad, usd.
+
+  @small_buttons: checkbock - Scales the buttons down.
+  @donation_amount_first: number. Required. The amount for the first button.
```

Comment:
Good catch here too. Fixed.

Question though, the button amounts are only optional as a group. It's all or none. I wasn't sure if I needed to explain that in the docs or not, but I can if you feel it's necessary.